<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>118</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>118</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#userFrame{
background-color:#2e8cff;
color:white;
font:14px;
padding:0px;
border-radius:20px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="userFrame">
     <property name="styleSheet">
      <string notr="true">*{
background-color:#2e8cff;
}
QPushButton#userPicture{
border:1px solid white;
border-radius:50px;
}
QPushButton#userPicture:hover{
border:2px solid white;
}
QPushButton#deleteUserBtn{
background-color:red;
border-radius:19px;

}
QPushButton#editUserBtn{
border-radius:20px;
background-color:green;
}
QPushButton#deleteUserBtn:hover{
border:2px solid white;
}
QPushButton#editUserBtn:hover{
border:2px solid white;
}
QPushButton#deleteUserBtn:pressed{
color: rgba(255, 0, 0,190);
border:3px solid orange;
}
QPushButton#editUserBtn:pressed{
background-color: rgba(0, 128, 0,190);
border:3px solid orange;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="userPicture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Full Name :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="fullNameLabel">
            <property name="text">
             <string>My Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Username :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="usernameLabel">
            <property name="text">
             <string>My Username</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Mobile No :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="mobileNoLabel">
            <property name="text">
             <string>+254700000000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Role :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="roleLabel">
            <property name="text">
             <string>Manager</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="editUserBtn">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteUserBtn">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
